models/vyos/ospf_v2/vyos_ospfv2.yaml

```diff
@@ -0,0 +1,331 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+        'metadata_version': '1.1',
+        'status': ['preview'],
+        'supported_by': 'network'
+    }
+
+NETWORK_OS: vyos
+RESOURCE: ospfv2
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: vyos_ospfv2
+  version_added: 2.10
+  short_description: This resource module configures and manages attributes of OSPFv2 routes on VyOS network devices.
+  description: This resource module configures and manages attributes of OSPFv2 routes on VyOS network devices.
+  author: Rohit Thakur (@rohitthakur2590)
+  options:
+    config:
+      description: A provided OSPFv2 route configuration.
+      type: dict
+      suboptions:
+        areas:
+          description: OSPFv2 area.
+          type: list
+          elements: dict
+          suboptions:
+            area_id:
+              description: Configured to discard packets.
```

Comment:
not sure what this description means.  Seems like the area_id the OSPF area ID (and that would be understood by any network user?)